cryptus-app/lib/collectionDictionary.ts

```diff
@@ -1,5 +1,5 @@
 const collections_dict = {
-  "0x1a92f7381b9f03921564a437210bb9396471050c": {
+  CoolCatsNFT: {
```

Comment:
keep addresses as references, but create name_clean in the fields. Addresses are unique but names are not, could create problems in the db

Reply:
If names can be the same, how are we gonna know which collection to show in the market overview? We want to have the collection name in the url, so if we have similar names, we can't use the collection name in the url. The context is used to search in the collection_dict...

Reply:
true I guess